build_osx.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# build_ubuntu.sh
+#
+# Build script for OSX using MacPorts, git pulls codec2 and
+# lpcnet repos so they are available for parallel development.
+
+export FREEDVGUIDIR=${PWD}
+export CODEC2DIR=$FREEDVGUIDIR/codec2
+export LPCNETDIR=$FREEDVGUIDIR/LPCNet
+export HAMLIBDIR=$FREEDVGUIDIR/hamlib
+
+# Prerequisite: build hamlib
+git clone git://git.code.sf.net/p/hamlib/code hamlib-code
+cd hamlib-code && git checkout master && git pull
+./bootstrap
+CFLAGS="-g -O2 -mmacosx-version-min=10.9" CXXFLAGS="-g -O2 -mmacosx-version-min=10.9" ./configure --disable-shared --prefix $HAMLIBDIR
+make
+make install
+
+# First build and install vanilla codec2 as we need -lcodec2 to build LPCNet
+cd $FREEDVGUIDIR
+git clone https://github.com/drowe67/codec2.git
+cd codec2 && git checkout master && git pull
```

Comment:
@tmiw do we need to clone from tmiw:master here to test this PR?

Reply:
Correct.

Reply:
I did this (by changing the script in my own local git repo), and it worked.